### PR TITLE
feat(retina): add disable network observability command for update

### PR DESCRIPTION
**Pull request by snguyen64**
add disable network observability command for update cluster and adds tests.
This was also mentioned in this issue: https://github.com/Azure/azure-cli-extensions/issues/6740
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az aks cluster update --disable-network-observability

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by FumingZhang — COMMENTED** [review]
lgtm

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=81806559&view=results) to validate the change. Test Passed!

Please resolve merge conflict.

**Review by FumingZhang — APPROVED** [review]

**Comment by yonzhan** [comment]
Please fix conflicting files

File: src/aks-preview/setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.166"
+VERSION = "0.5.167"
```
Comment (zhoxing-ms) [inline src/aks-preview/setup.py:12]: The extension version  `0.5.167` is already exists

Comment (FumingZhang) [reply src/aks-preview/setup.py]: resolved, thanks

**Comment by yanzhudd** [comment]
please fix CI failed items

**Comment by FumingZhang** [comment]
Fixed the failed test and resolved conflict in PR #6951

**Pull request closed without merging**